public/output/output.js

```diff
@@ -126,14 +126,11 @@ function scoreboard() { // generate the scoreboard in right column div with ID '
     // dynamically generate user area
     userDiv = select('#users');
     userDiv.html(addUsers());
-  } else {
-    userDiv = select('#users');
   }
-
 }
 
 function addUsers() {
-  let output;
+  let output = '';
```

Comment:
@ntwallace this fixes the `undefined` problem you were seeing.

![image](https://user-images.githubusercontent.com/478136/37051787-6e8ef634-2145-11e8-9974-79eef150d8ac.png)